src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -373,9 +373,11 @@ private void TagGroupHelper(List<Channel.TagGroupsEditor.TagOperation> payload,
 
         public bool OnDeepLink(string deepLink)
         {
-            var handler = onDeepLinkReceived;
-            handler(deepLink);
-            return true;
+            if (onDeepLinkReceived != null) {
```

Comment:
handlers should always be checked for null. the easy way to do this in one line in c# is `handler?.Invoke(method)`, but in this case we also want to return a different result depending on whether we handled the link or not